src/library/utils/rtlTextAlign.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+
+export default function rtlTextAlign(align?: string, direction?: string) {
+  const rtl = direction === 'rtl';
+  if ((rtl && align == 'start') || (!rtl && align == 'end')) {
+    return 'right';
+  } else if ((rtl && align == 'end') || align == 'start') {
```

Comment:
this looks ripe for a unit test.

also, why double equal and not triple?

Reply:
Done and fixed.